DuckDuckGo/TabViewController.swift

```diff
@@ -333,7 +333,9 @@ class TabViewController: WebViewController {
     }
     
     private func openExternally(url: URL) {
-        UIApplication.shared.open(url, options: [:])
+        UIApplication.shared.open(url, options: [:]) { completion in
+            self.view.showBottomToast(UserText.failedToOpenExternally)
```

Comment:
This displays even when an app can handle the scheme. There's some confusion here. The parameter named `completion` is a bool indicating success or failure. Maybe rename it to `success` and then only display the toast if `!success`?